addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1564,7 +1565,10 @@ const Wysiwyg = Widget.extend({
                         }
                     });
                     colorpicker.on('color_leave', null, ev => {
-                        this.odooEditor.historyRevertCurrentStep();
+                        if (!this._colorPicked) {
+                            this.odooEditor.historyRevertCurrentStep();
+                        }
+                        this._colorPicked = false;
```

Comment:
The color feature itself works while picking the color, even though the current step is reverted. So, somehow, the color itself is not reverted. What is the bug that this is supposed to fix ?

Reply:
It's strange I wasn't able to reproduce. But since a bunch of things were fixed in the colorpicker recently, maybe the bug I meant to fix here a while ago doesn't occur anymore, I don't know. Reverting.